* fixes including of <linux/rtc.h> and <linux/input.h> from userland (MPlayer)

diff -Nur linux-2.6.16.orig/include/linux/input.h linux-2.6.16/include/linux/input.h
--- linux-2.6.16.orig/include/linux/input.h	Mon Mar 20 07:53:29 2006
+++ linux-2.6.16/include/linux/input.h	Sat Mar 25 12:50:00 2006
@@ -796,6 +796,8 @@
 
 #define FF_MAX		0x7f
 
+#ifdef BITS_PER_LONG
+
 struct input_device_id {
 
 	kernel_ulong_t flags;
@@ -814,6 +816,8 @@
 
 	kernel_ulong_t driver_info;
 };
+
+#endif
 
 /*
  * Structure for hotplug & device<->driver matching.
diff -Nur linux-2.6.16.orig/include/linux/rtc.h linux-2.6.16/include/linux/rtc.h
--- linux-2.6.16.orig/include/linux/rtc.h	Mon Mar 20 07:53:29 2006
+++ linux-2.6.16/include/linux/rtc.h	Sat Mar 25 14:28:59 2006
@@ -11,7 +11,9 @@
 #ifndef _LINUX_RTC_H_
 #define _LINUX_RTC_H_
 
+#ifdef __KERNEL__
 #include <linux/interrupt.h>
+#endif
 
 /*
  * The struct used to pass data via the following ioctl. Similar to the
